nextflow_schema.json

```
@@ -1,47 +1,520 @@
+                    "description": "'fitType' param passed to DESeq()",
+                    "help_text": "either \"parametric\", \"local\", \"mean\", or \"glmGamPoi\" for the type of fitting of dispersions to the mean intensity. See estimateDispersions for description."
+                },
+                "deseq2_sf_type": {
+                    "type": "string",
+                    "default": "ratio",
+                    "description": "'sfType' param passed to DESeq()",
+                    "help_text": "either \"ratio\", \"poscounts\", or \"iterate\" for the type of size factor estimation. See estimateSizeFactors for description."
+                },
+                "deseq2_min_replicates_for_replace": {
+                    "type": "integer",
+                    "default": 7,
+                    "help_text": "the minimum number of replicates required in order to use replaceOutliers on a sample. If there are samples with so many replicates, the model will be refit after these replacing outliers, flagged by Cook's distance. Set to Inf in order to never replace outliers. It set to Inf for fitType=\"glmGamPoi\".",
+                    "description": "'minReplicatesForReplace' param passed to DESeq()"
+                },
+                "deseq2_use_t": {
+                    "type": "boolean",
+                    "description": "'useT' param passed to DESeq2",
+                    "help_text": "logical, passed to nbinomWaldTest, default is FALSE, where Wald statistics are assumed to follow a standard Normal"
+                },
+                "deseq2_independent_filtering": {
+                    "type": "boolean",
+                    "default": true,
+                    "description": "'independentFiltering' param passed to results()",
+                    "help_text": "logical, whether independent filtering should be applied automatically"
+                },
+                "deseq2_lfc_threshold": {
+                    "type": "integer",
+                    "default": 0,
+                    "description": "'lfcThreshold' param passed to results()",
+                    "help_text": "a non-negative value which specifies a log2 fold change threshold. The default value is 0, corresponding to a test that the log2 fold changes are equal to zero. The user can specify the alternative hypothesis using the altHypothesis argument, which defaults to testing for log2 fold changes greater in absolute value than a given threshold. If lfcThreshold is specified, the results are for Wald tests, and LRT p-values will be overwritten."
+                },
+                "deseq2_alt_hypothesis": {
                     "type": "string",
-                    "description": "MultiQC report title. Printed as page header, used for filename if not otherwise specified.",
-                    "fa_icon": "fas fa-file-signature"
+                    "default": "greaterAbs",
+                    "description": "'altHypothesis' param passed to results()",
```

```suggestion
                    "description": "`altHypothesis` param passed to results()",
```
Change this also in all following descriptions